lib/shared/sdk/usbboot/usb.js

```diff
@@ -67,7 +68,26 @@ const USB_CONTROL_TRANSFER_TIMEOUT_MS = 0
  * })
  */
 exports.listDevices = () => {
-  return Bluebird.resolve(usb.getDeviceList())
+  const devices = _.map(usb.getDeviceList(), (device) => {
+    device.driver = true
+    return device
+  })
+
+  // Include driverless devices into the list of USB devices
+  if (os.platform() === 'win32') {
+    const winusbDriverGenerator = require('winusb-driver-generator')
+    _.each(winusbDriverGenerator.listDriverlessDevices(), (device) => {
+      devices.push({
```

Comment:
I guess we probably want to filter this list of all DriverlessDevices to just those VIDs and PIDs that we're interested in :stuck_out_tongue_winking_eye:

Reply:
We do the filtering on the usbboot side. This is a "usb" module that doesn't care about the things it lists.

Reply:
Ah, my mistake! I initially saw:
```
      if (!device.accessible) {
        return {
          device: `${usbIdToString(device.deviceDescriptor.idVendor)}:${usbIdToString(device.deviceDescriptor.idProduct)}`,
          displayName: 'Missing drivers',
          description: 'Compute Module',
```
in `lib/shared/sdk/usbboot/index.js` and thought you were assuming that any driverless device was a 'Compute Module'. I'd missed that you're also doing a `filter(isUsbBootCapableUSBDevice)` just a few lines above ;-)